controllers/vgmanager_daemonset.go

```diff
@@ -189,3 +207,31 @@ func newVGManagerDaemonset(lvmCluster lvmv1alpha1.LVMCluster) appsv1.DaemonSet {
 	setDaemonsetNodeSelector(nodeSelector, &ds)
 	return ds
 }
+
+func getRunningPodImage(r *LVMClusterReconciler, ctx context.Context) (string, error) {
```

Comment:
how about adding the image as env in the csv?

Reply:
- that's also and option, we need to update both image and env variable in that case.